homestar-wasm/src/wasmtime/world.rs

```diff
@@ -145,6 +146,7 @@ impl<T> Env<T> {
     /// Types must conform to [Wit] IDL types when Wasm was compiled/generated.
     ///
     /// [Wit]: <https://github.com/WebAssembly/component-model/blob/main/design/mvp/WIT.md>
+    #[instrument(name = "execute", skip_all)]
```

Comment:
minor: does this need the name since it matches the fn? I thought that was automatic, but maybe not?